code/go/0chain.net/smartcontract/zcnsc/handler.go

```diff
@@ -2,24 +2,52 @@ package zcnsc
 
 import (
 	"context"
-	"errors"
 	"net/url"
 
 	"0chain.net/smartcontract"
 
+	"0chain.net/smartcontract/dbs/event"
+
 	cState "0chain.net/chaincore/chain/state"
+	"github.com/pkg/errors"
 )
 
-func (zcn *ZCNSmartContract) getAuthorizerNode(_ context.Context, params url.Values, ctx cState.StateContextI) (interface{}, error) {
-	authorizerID := params.Get("id")
-	if authorizerID == "" {
-		return nil, errors.New("authorizerID is empty")
+type AuthorizerNodeResponse struct {
+	ID  string `json:"id"`
+	URL string `json:"url"`
+}
+
+func (zcn *ZCNSmartContract) getAuthorizerNodes(
```

Comment:
In anticipation of https://github.com/0chain/0chain/issues/926 we could prefix new endpoints with  goDoc quality comments.

Reply:
fixed